log4j-core/src/main/java/org/apache/logging/log4j/core/appender/rolling/RollingRandomAccessFileManager.java

```diff
@@ -156,7 +156,9 @@ public synchronized boolean closeOutputStream() {
         flush();
         if (randomAccessFile != null) {
             try {
-                randomAccessFile.close();
+                if (randomAccessFile != null) {
+                    randomAccessFile.close();
+                }
```

Comment:
I think this was recently fixed on line 157, see https://issues.apache.org/jira/browse/LOG4J2-2418

Reply:
True. My bad, couldn't find that one. Closing.

Reply:
No worries, thank you for taking the time to submit a change upstream!